### DISCUSSION TITLE
"columns" parameter in dropna

### DISCUSSION THREAD

**Discussion by Teut2711** [opensource9ja/danfojs#312] [Q&A] [answered]
Currently it removes all rows where value in any column is null. Can an argument of "columns" be provided to check null in specific columns only?

https://danfo.jsdata.org/api-reference/dataframe/danfo.dataframe.dropna

For my dataframe, I have to use `df.loc` to achieve that like this
```


        testsDf.loc({
            rows: testsDf["Location"].isna().eq(false).and(
                testsDf["Dept"].isna().eq(false).and(
                    testsDf["Gender"].isna().eq(false).
                        testsDf["Age"].isna().eq(false).and(
                            
                        )
```

**Comment by risenW** [comment, marked as answer]
@Teut2711 At the moment, no it doesn't support a column parameter. But this looks useful, it can be added.
